rules/windows/image_load/image_load_side_load_from_non_system_location.yml

```diff
@@ -451,51 +451,52 @@ detection:
         ImageLoaded|endswith: '\cscui.dll'
     filter_main_defender:
-        ImageLoaded|startswith: 'C:\ProgramData\Microsoft\Windows Defender\Platform\'
+        ImageLoaded|contains: ':\ProgramData\Microsoft\Windows Defender\Platform\'
         ImageLoaded|endswith: '\version.dll'
     filter_main_directx:
-        ImageLoaded|startswith: 'C:\Program Files\WindowsApps\Microsoft.DirectXRuntime_'
+        ImageLoaded|contains: ':\Program Files\WindowsApps\Microsoft.DirectXRuntime_'
         ImageLoaded|endswith: '\d3dx9_43.dll'
     filter_optional_exchange:
-        ImageLoaded|startswith: 'C:\Program Files\Microsoft\Exchange Server\'
+        ImageLoaded|contains: ':\Program Files\Microsoft\Exchange Server\'
         ImageLoaded|endswith: '\mswb7.dll'
     filter_optional_arsenal_image_mounter:
-        ImageLoaded|startswith: 'C:\Program Files\Arsenal-Image-Mounter-'
+        ImageLoaded|contains: ':\Program Files\Arsenal-Image-Mounter-'
         ImageLoaded|endswith:
             - '\mi.dll'
             - '\miutils.dl'
     filter_optional_office_appvpolicy:
-        Image: 'C:\Program Files\Common Files\microsoft shared\ClickToRun\OfficeClickToRun.exe'
-        ImageLoaded: 'C:\Program Files\Common Files\microsoft shared\ClickToRun\AppVPolicy.dll'
+        Image|contains: ':\Program Files\Common Files\microsoft shared\ClickToRun\OfficeClickToRun.exe'
+        ImageLoaded: ':\Program Files\Common Files\microsoft shared\ClickToRun\AppVPolicy.dll'
     filter_optional_azure:
-        ImageLoaded|startswith: 'C:\Packages\Plugins\Microsoft.GuestConfiguration.ConfigurationforWindows\'
+        ImageLoaded|contains: ':\Packages\Plugins\Microsoft.GuestConfiguration.ConfigurationforWindows\'
     filter_optional_dell:
         Image|contains:
-            - 'C:\Program Files\WindowsApps\DellInc.DellSupportAssistforPCs'
-            - 'C:\Windows\System32\backgroundTaskHost.exe'
-        ImageLoaded|startswith: 'C:\Program Files\WindowsApps\DellInc.DellSupportAssistforPCs'
+            - ':\Program Files\WindowsApps\DellInc.DellSupportAssistforPCs'
+            - ':\Windows\System32\backgroundTaskHost.exe'
+        ImageLoaded|contains: ':\Program Files\WindowsApps\DellInc.DellSupportAssistforPCs'
     filter_optional_dell_wldp:
-        Image|startswith: 'C:\Program Files\WindowsApps\DellInc.DellSupportAssistforPCs'
+        Image|contains: ':\Program Files\WindowsApps\DellInc.DellSupportAssistforPCs'
         Image|endswith: '\wldp.dll'
     filter_optional_checkpoint:
-        Image|startswith:
-            - 'C:\Program Files\CheckPoint\'
-            - 'C:\Program Files (x86)\CheckPoint\'
+        Image|contains:
+            - ':\Program Files\CheckPoint\'
+            - ':\Program Files (x86)\CheckPoint\'
         Image|endswith: '\SmartConsole.exe'
-        ImageLoaded|startswith:
-            - 'C:\Program Files\CheckPoint\'
-            - 'C:\Program Files (x86)\CheckPoint\'
+        ImageLoaded|contains:
```

Comment:
Change this back for all as filter needs to be exact

Reply:
A filter is an exclusion and a broad exclusion create false negatives. For examples the `C:\Program Files\CheckPoint\` path is part of the legit `Program Files` of the OS, and `D:\Program Files\CheckPoint` is not and should be matched on, or excluded by the user.